scripts/setup_host.sh

```diff
@@ -13,6 +13,7 @@ QEMU_REL="qemu-4.2.0"
 CIV_WORK_DIR=$(pwd)
 CIV_GOP_DIR=$CIV_WORK_DIR/GOP_PKG
 CIV_VERTICAl_DIR=$CIV_WORK_DIR/vertical_patches/host
+CIV_VERTICAl_PROD_DIR=$CIV_WORK_DIR/vertical_prod_patches/host/caas_tpy
```

Comment:
could you add more commit message to explain why need this patch?
what's the difference between CIV_VERTICAl_DIR and CIV_VERTICAl_PROD_DIR?

Reply:
@YadongQi : 
CIV_VERTICAl_DIR: Which contains common vertical Patches for the releases
CIV_VERTICAl_PROD_DIR:  Which Contains Customer Specific Patches for releases which we are creating new branch of [celedon/r/tpy/vertical] for cust specific releases 
While running host script it will fetch host patches form both the DIR